src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
+{
+  console.log("You have clicked the page");
+  write();
+  var date = new Date();
+  dateTime(date);
+}
+
+function write()
+{
+  const para = document.createElement("p");
+  para.innerText = "You have clicked the page!";
+  document.body.appendChild(para);
+
+  var dt = new Date();
+  const datetime = document.createElement("p");
+  datetime.innerHTML = dt;
+  document.body.appendChild(datetime);
+
+  document.getElementById('date-time').innerHTML = dt;
+}
+
+function dateTime(date:Date):void
```

Comment:
nice job using typescript to mark that this is a `Date` object! Similar to above, can you convert it to an es6 function?

Reply:
Converted this function to an arrow function